tests/test_modeling.py

```
@@ -13,370 +13,325 @@
-        assert prompts_mask.shape == (prompt_setup['batch_size'], 3)
-    
+        assert prompts_embedding.shape == (prompt_setup["batch_size"], 3, prompt_setup["embed_dim"])
+        assert prompts_mask.shape == (prompt_setup["batch_size"], 3)
+
     def test_extracts_class_token_embeddings(self, prompt_setup):
         """Should extract embeddings at class token positions when embed_ent_token=True."""
-        prompts_embedding, prompts_mask = extract_prompt_features(
-            **prompt_setup, embed_ent_token=True
-        )
-        
+        prompts_embedding, prompts_mask = extract_prompt_features(**prompt_setup, embed_ent_token=True)
```

Ruff (0.15.1)

[warning] Unpacked variable `prompts_mask` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)